Prog/Cvd19_affctd_ind_pop.sas

```diff
@@ -0,0 +1,254 @@
+/**************************************************************************
+ Program:  Cvd19_affctd_ind_pop.sas
+ Library:  Covid19
+ Project:  Urban-Greater DC
+ Author:   P. Tatian
+ Created:  03/28/20
+ Version:  SAS 9.4
+ Environment:  Local Windows session (desktop)
+ GitHub issue:  1
+ 
+ Description:  Describe characteristics of population in COVID-19
+ affected industries: transportation, hospitality, mining, travel
+ using ACS IPUMS data. 
+
+ Modifications:
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( Covid19 )
+%DCData_lib( Ipums )
+
+%let RegPumas = "1100101", "1100102", "1100103", "1100104", "1100105", "2401600", "2400301", "2400302","2401001", "2401002", 
+				"2401003", "2401004", "2401005", "2401006", "2401007", "2401101", "2401102", "2401103", "2401104", "2401105", 
+				"2401106", "2401107", "5101301", "5101302", "5159301", "5159302", "5159303", "5159304", "5159305", "5159306", 
+				"5159307", "5159308", "5159309", "5110701", "5110702" , "5110703", "5151244", "5151245", "5151246", "5151255" ;
+
+%let keep_vars = 
+  serial pernum numprec perwt hhwt ind occ year statefip countyfip met2013 upuma
+  age sex 
+  raced racwht racblk racasian racamind racpacis racother hispand
+  educd school schltype
+  hud_inc poverty inc: 
+  labforce empstatd wkswork2 uhrswork tranwork classwkrd
+  ownershp ownershpd rentgrs owncost gq plumbing hotwater;
+
+data A;
+
+  set
+    Ipums.Acs_2014_18_dc (keep=&keep_vars)
+    Ipums.Acs_2014_18_md (keep=&keep_vars)
+    Ipums.Acs_2014_18_va (keep=&keep_vars)
+    Ipums.Acs_2014_18_wv (keep=&keep_vars);
+
+  where met2013 = 47900;
+  
+  retain total 1;
+
+  if upuma in (&RegPumas.) then MWCOG_region = 1;
+  else MWCOG_region = 0;
+  
+  /* Flag 35+ hours worked as full time */
+  if uhrswork >= 35 then fulltime=1;
+  else if uhrswork > 0 then fulltime=0;
+  else fulltime = .n;
+
+  /* Flag 50-52 weeks per year as year-round */
+  if wkswork2 = 6  then yearround=1;
+  else if wkswork2 > 0 then yearround=0;
+  else yearround = .n;
+
+  if 9920 > ind > 0 then do; 
+  
+    select;
+    
+      when ( year in ( 2018 ) )
```

Comment:
I assume it's code this way because the industry codes over time, but since  the codes for industries affected by covid-19 are the same in all years, the when statements are redundant.